backend/django/core/data/SemEval-2016-Task6/test-awww.csv

```diff
@@ -0,0 +1,72 @@
+Text,Label

```

Comment:
One thing I didn't notice earlier, the test-aww.csv file should go in a new folder `backend/django/core/data/reddit_sample_awww`. The semeval folder is specifically for data to do with the SemEval data challenge which is the source of the feminism tweet data.

Reply:
done